packages/react-components/src/components/table-output-component.tsx

```diff
@@ -421,6 +428,9 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 ['table_search_expressions']: filterObj,
                 ['isFiltered']: isFiltered
             };
+            if (direction !== undefined) {
+                additionalParams['table_search_direction'] = direction.toString;
```

Comment:
This actually puts the toString function reference in the map ;) it's missing () after toString...

Reply:
good catch